### ISSUE TITLE
Allow merging of table cells

### ISSUE THREAD

**Issue by michaelglasson** [bpmn-io/dmn-js#836]
### Allow adjacent cells in a column to be merged

The DMN Specification shows that it is valid to merge adjacent cells so that multiple cells can have the same content which needs to be entered only once.

### Describe alternatives you've considered

Putting the same data into adjacent cells - as is the case now.

### Additional context

There are many examples in the DMN specification that show merged cells.

**Comment by barmac** [comment]
Thanks for creating this issue. How do you imagine such solution could work from the user's point of view? What actions should I take to merge the table cells?

**Comment by barmac** [comment]
Makes sense. So the hidden requirement of this feature request is the ability to select multiple cells. This could unlock even more actions, e.g. clear multiple cells at once. Anyway, for now this lands in backlog so that we can pick it up later.
